source/Cute.Lib/Serializers/EntrySerializer.cs

```diff
@@ -15,7 +15,21 @@ public class EntrySerializer
 
     private readonly string[] _locales;
 
-    public IEnumerable<string> ColumnFieldNames => _sysFields.Concat(_fieldSerializers.Keys);
+    public static readonly HashSet<string> SysFields = [
```

Comment:
Maybe make "get" accessor public, and remove ability to "set". Even though readonly the HashSet can still be added/removed from. Consider returning as IReadOnlyCollection<> if that works upstream.